src/store/searches-slice.ts

```diff
@@ -51,51 +62,54 @@ const fetchSearchesFulfilled = (
 		state.more = payload?.hasMore === true;
 		state.offset = (meta?.arg.offset ?? 0) + 100;
 		state.sortBy = meta?.arg.sortBy ?? 'dateDesc';
+		state.error = undefined;
 	}
 };
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const fetchSearchesRejected = (state: SearchesStateType, { payload }): void => {
+	state.status = 'error';
+	state.conversations = [];
+	state.messages = [];
+	state.error = {
+		code: payload?.Detail.Error.Code ?? 'generic error'
+	};
+};
+
 const msgActionFulfilled = (
 	state: SearchesStateType,
 	{ meta }: { meta: { arg: ConvActionParameters; requestId: string; requestStatus: string } }
 ): void => {
 	if (meta.arg.ids) {
-		// state.conversations = state.conversations
-		// 	? map(state.conversations, (conv) => ({
-		// 			...conv,
-		// 			messages: map(conv.messages, (msg) => {
-		// 				if (includes(meta.arg.ids, msg.id)) {
-		// 					return { ...msg, parent: FOLDERS.TRASH };
-		// 				}
-		// 				return msg;
-		// 			})
-		// 	  }))
-		// 	: [];
-		// state.messages = state.messages
-		// 	? map(state.messages, (msg) => {
-		// 			if (includes(meta.arg.ids, msg.id)) {
-		// 				return { ...msg, parent: FOLDERS.TRASH };
-		// 			}
-		// 			return msg;
-		// 	  })
-		// 	: [];
+		state.error = undefined;
```

Comment:
I have a little doubt regarding this ,I think it is not needed because I remember I added reducers to sync it with the notify so if there are changes we will receive it in the notify and it will automatically be sync. (I may be wrong because I'm not testing it by running it locally)